From b2e2e29d048405766c2c19d73d6122b753ba65ef Mon Sep 17 00:00:00 2001
From: Dominik Menke <dom@digineo.de>
Date: Tue, 4 Feb 2020 14:29:31 +0100
Subject: [PATCH] Change default login credentials

The router has an adhesive label with some default login credentials,
which were valid for the LEDE era. Sometime after we've ordered the
devices, upstream changed to an empty password by default.

This changes the root password back to the printed value.

Signed-off-by: Dominik Menke <dom@digineo.de>
---
 package/base-files/files/etc/shadow | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/base-files/files/etc/shadow b/package/base-files/files/etc/shadow
index 4b4154f21f..b18314fc94 100644
--- a/package/base-files/files/etc/shadow
+++ b/package/base-files/files/etc/shadow
@@ -1,4 +1,4 @@
-root::0:0:99999:7:::
+root:$1$L6URLL4o$cWPDFK1OTa0d3j28aeiIs0:0:0:99999:7:::
 daemon:*:0:0:99999:7:::
 ftp:*:0:0:99999:7:::
 network:*:0:0:99999:7:::
-- 
2.20.1

